Recentering

Humungus-Fungus opened the pull request:
Pressing space now centers cam on champ instead of the center of the map.
Pressing Y toggles this.

noah1510 commented:
You will have to reimplement this after #13 is merged. Both of you properly reformatted the player script but the other pr is for the whole project.

noah1510 commented:
Also all camera controls should be purely client only. Why is there server communication in camera movement? The clients don't need the server permission to move their camera where they want it to be.

Humungus-Fungus commented:
Updated to work with #13 and now uses the names of the champion nodes to get their pid instead of asking the server.

Humungus-Fungus commented:
Oops, didn't even notice the change in names. Reworked center_cam. Changed is_cam_centered into a config setting, now you have the option to change the center toggle through settings or with Y, both work.

noah1510 commented on Scripts/config.gd:
```diff
@@ -59,3 +60,8 @@ func set_min_zoom(new_value: float):
 	camera_property_changed.emit()
+
+
+func set_cam_centered(new_value: float):
```
should be bool

noah1510 commented:
Besides the wrong type this is good to go
